userspace/libsinsp/sinsp_filter_transformers/sinsp_filter_transformer.cpp

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2025 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <libsinsp/sinsp_filter_transformers/sinsp_filter_transformer.h>
+
+sinsp_filter_transformer::~sinsp_filter_transformer() {}
+
+bool sinsp_filter_transformer::transform_type(ppm_param_type& t, uint32_t& flags) const {
```

Comment:
What if we drop this implementation in the base class by making it a pure virtual method?

Reply:
The same you did for `transform_values()`